Author: Sven Joachim <svenjoac@gmx.de>
Description: Debian changes to xterm* terminfo entries
 The xterm-debian terminfo entry is provided as an alias for xterm, as
 some ancient Debian systems used xterm-debian as $TERM.
Forwarded: not-needed
Last-Update: 2023-11-19

---
 misc/terminfo.src |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -5667,7 +5667,7 @@ xterm-24|vs100|xterms|xterm terminal emu
 	lines#24, use=xterm-old,
 
 # This is xterm for ncurses.
-xterm|xterm terminal emulator (X Window System),
+xterm|xterm-debian|xterm terminal emulator (X Window System),
 	use=xterm-new,
 
 # This entry assumes that xterm's handling of VT100 SI/SO is disabled by
